lightcone/animate.py

```diff
@@ -104,6 +108,10 @@ def save():
             os.system(os_string)
 
         save()
+        movie_file = self.movie_name()
+        file_exists = exists(movie_file)
+        if not file_exists:
+            print("WARNING!: MP4 File not detected. Make sure you have FFMPEG installed and in your PATH.")
```

Comment:
there are also dedicated WARNING() functionalities, such that you do not need a print() statement in that form